README

```diff
@@ -12,4 +12,17 @@ autoreconf -vfi
 
 ./configure && make && make install
 
+Building enet -use vcpkg
```

Comment:
This feels like it should be a headline. The wording is also weird: `-use vcpkg`?
I also feel like this should be "Installing enet (vcpkg)" because it doesn't just build it - it also sort-of adds it to an index / central location.